foundation/foundation-configuration-store/src/main/java/com/boozallen/aissemble/configuration/mutating/webhook/ConfigMutatingWebhook.java

```diff
@@ -129,9 +151,63 @@ private String getProperty(String groupName,String propertyName) {
         }
     }
 
+    //This method is to inject values in the data fields for any encoded Configvalues from the properties file and save it as based64 encoded.
+    private String replaceSecretConfigStoreKeyWithConfigValue(String json, ObjectMapper mapper) throws JsonProcessingException {
+
+        //inject any plain text configuration store values first.
+        String plainJsonInjection = replaceConfigStoreKeysWithConfigValue(json);
+
+        String updatedJSON = null;
+
+        if(!plainJsonInjection.isEmpty())
+        {
```

Comment:
A: This doesn't match the standard code styling of the project.
```suggestion
        if (!plainJsonInjection.isEmpty()) {
```